src/web/components/LayoutSlotBottom.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { css } from 'emotion';
+import { useApi } from '@root/src/web/components/lib/api';
+
+const wrapperMargins = css`
+    margin: 18px 0;
+`;
+
+export const LayoutSlotBottom = () => {
+    const endpointUrl =
+        'https://zsxulafpt1.execute-api.eu-west-1.amazonaws.com/prod';
+    const { data, error } = useApi(endpointUrl);
+
+    if (error) {
+        window.guardian.modules.sentry.reportError(error, 'layout-slot-bottom');
+        return null;
+    }
+
+    if (data) {
```

Comment:
Do you need to protect against `data.css` and `data.html` being undefined too?

Reply:
Don't think we do, we'd just end up with an empty slot with no HTML or CSS. But I've added some checks anyway to prevent either from being added to the DOM unless they have any content.